numba/_helperlib.c

```diff
@@ -19,7 +19,13 @@
 #else
     #include <stdint.h>
     #define _complex_float_t complex float
-    #define _complex_float_ctor(r, i) (r + I * i)
+    #if defined(_Imaginary_I)
+        #define _complex_float_ctor(r, i) (r + _Imaginary_I * i)
+    #elif defined(_Complex_I)
+        #define _complex_float_ctor(r, i) (r + _Complex_I * i)
+    #else
+        #error "Lack _Imaginery_I and _Complex_I"
```

Comment:
A small typo that might have a user grepping for something that will never be found:

```suggestion
        #error "Lack _Imaginary_I and _Complex_I"
```